learning_journal/learning_journal/tests/test_models.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+from learning_journal.models import Entry, DBSession
+
+
+def test_create_mymodel(dbtransaction, dummy_request):
+    """Test creation of model."""
+    new_model = Entry(title="Norton", text="waffles")
+    assert new_model.id is None
+    DBSession.add(new_model)
+    DBSession.flush()
+    assert new_model.id is not None
```

Comment:
Is there anything else you can test about your model?  Is this the only functionality it has that you want to test?  It certainly shouldn't be